src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -522,7 +538,6 @@ public function get_theme_item_permissions_check( $request ) {
 	 * @since 5.9.0
 	 *
 	 * @param WP_REST_Request $request The request instance.
-	 *
```

Comment:
Why was this line removed

Reply:
Coding standard of no newlines between the params and return. A wee bit of girl scouting.